.idea/gradle.xml

```diff
@@ -16,6 +16,18 @@
           </set>
         </option>
       </GradleProjectSettings>
+      <GradleProjectSettings>
+        <option name="distributionType" value="DEFAULT_WRAPPED" />
+        <option name="externalProjectPath" value="$PROJECT_DIR$/code_samples/editor_basics" />
+        <option name="gradleHome" value="C:/Program Files (x86)/Gradle/gradle-2.9" />
```

Comment:
You're referring to the `value` of the `gradleHome` option?  I can change it, but...

This doesn't have meaning - at least not on my machine. As I have added Gradle plugins this value has been added by IntelliJ. It probably propagated (from gradle_plugin_demo) even though my GRADLE_HOME setting was actually used. I view this setting like the `project-jdk-name` in misc.xml, a placeholder that the user has to set anyway.

Reply:
there shouldn't be a `gradleHome` entry at all since the IDE should use Gradle defined in /gradle/wrapper inside project and not some local installation

Reply:
As I have added Gradle-based plugins to the SDK using 2019.1, the Gradle Home option element value in gradle.xml:
<option name=“gradleHome” value=“C:/Program Files (x86)/Gradle/gradle-2.9" />
propagated to each added plugin, despite doing all the work on a Mac.
I believe this is because the original Gradle-based plugin in the project had this value in ‘gradle.xml’, and it populated the “Gradle home” preference in 2019.1. (Even though “Use local Gradle distribution” wasn’t enabled.)
As I work on these plugins in 2019.2 the value isn’t changed in gradle.xml, despite the setting “Use Gradle from: ‘gradle-wrapper.properties’ file.” The gradleHome value is retained in 2019.2 to support the “Use Gradle from: Specified location” preference.

The option has been removed by choosing the “Use Gradle from: Specified location” pref, clear the location field (pre-populated with the Win path,) apply, choose the “Use Gradle from: ‘gradle-wrapper.properties’ file” pref, apply.